src/main/java/ch/njol/skript/entity/MinecartData.java

```diff
@@ -52,12 +46,14 @@ public String toString() {
 		static {
 			final ArrayList<String> cn = new ArrayList<>();
 			for (final MinecartType t : values()) {
-				if (t.c != null)
+				if (t.entityClass != null)
 					cn.add(t.codeName);
 			}
 			codeNames = cn.toArray(new String[0]);
```

Comment:
```suggestion
			CODENAMES = cn.toArray(new String[0]);
```
since its public static and isnt changed after init, can use upper case